docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,136 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Getting Started
+#
+# This document provides an overview for preparing to work with PDO contracts through the contract
+# docker container. It will walk through the necessary steps to set up a functional configuration.
+#
+# %% [markdown]
+# ## How Contracts Use Jupyter
+#
+# Many of the PDO contract families provide Jupyter notebooks to simplify interaction with contract
+# objects. Commonly, the notebooks provide a factory for instantiating a notebook for each new
+# contract from one of the contract specific templates.The templates provided generally share five
+# common sections:
+#
+# * Configure the Contract Object
+# * Initialize the Interpreter and the PDO Environment
```

Comment:
1. not sure what is meant by configure the contract object (specifically, how is this related to initialize contract object context)
2. initialize the interpret -> you mean initialize the client?

Reply:
please suggest wording changes.

Reply:
Section rewritten. However, this really is about the Jupyter interpreter and the PDO Environment.